pkg/model/stack.go

```diff
@@ -2,49 +2,19 @@ package model
 
 import (
 	"fmt"
-	"time"
-
-	"gorm.io/gorm"
 )
 
 // swagger:model Stack
 type Stack struct {
-	Name      string    `json:"name" gorm:"primaryKey"`
-	CreatedAt time.Time `json:"createdAt"`
-	UpdatedAt time.Time `json:"updatedAt"`
-	// GormParameters are only used by Gorm to persist parameters as it cannot persist a
-	// StackParameters. Only use GormParameters within the repository. Otherwise use
-	// Parameters.
-	GormParameters []StackParameter `json:"parameters" gorm:"foreignKey:StackName; references: Name; constraint:OnUpdate:CASCADE,OnDelete:CASCADE"`
-	// Parameters used by the stacks helmfile template.
-	Parameters       StackParameters `json:"-" gorm:"-"`
-	Instances        []Instance      `json:"instances" gorm:"constraint:OnUpdate:CASCADE,OnDelete:CASCADE;"`
+	Name             string          `json:"name"`
+	Parameters       StackParameters `json:"parameters"`
```

Comment:
You are introducing a breaking change here going from `[]` to a `{}` of parameters. Is that intentional? `{}` is what I prefer to have in the end 😋 just checking